app/components/listing/ResourceInfos.jsx

```diff
@@ -89,7 +89,7 @@ AddressInfo.propTypes = {
 
 const PhoneNumber = ({ phones }) => (
   <span className="phone">
-    { phones.map(phone => <p key={phone.id}>{`${phone.number} ${phone.service_type}`}</p>) }
+    { phones.map(phone => <p key={phone.id}><a href={`tel:${phone.number}`}>{`${phone.number}`}</a>{` ${phone.service_type}`}</p>) }
```

Comment:
Thanks! I have one last code change suggestion. The (` `` `) syntax in JavaScript (known as [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)) allows you to define a string where you interpolate the value of a variable into the string. It doesn't make much sense to use it if the only in the string is the variable itself, so it'd be simpler to just return the variable in those cases. Specifically, for this line of code, I would suggest writing it like the following:

```suggestion
    { phones.map(phone => <p key={phone.id}><a href={`tel:${phone.number}`}>{phone.number}</a>{` ${phone.service_type}`}</p>) }
```